linkcheck/checker/httpurl.py

```diff
@@ -78,15 +78,11 @@ def content_allows_robots (self):
         """
         if not self.is_html():
             return True
-        # construct handler object
-        handler = linkparse.MetaRobotsFinder()
-        # parse
-        try:
-            htmlsoup.process_soup(handler, self.get_soup())
-        except linkparse.StopParse as msg:
-            log.debug(LOG_CHECK, "Stopped parsing: %s", msg)
-            pass
-        return handler.follow
+
+        if self.get_soup().find("meta", attrs={"name": "robots",
+                                               "content": "nofollow"}):
```

Comment:
interesting. i hadn't noticed that code when i submitted a patch to skip "robots.txt" (9d899d1d) - maybe that should respect the `--no-robots` flag as well? but that's not new to this change so i guess it's just a note to self...

Reply:
And even where the check should go, perhaps the existing one should be in HttpUrl.check_connection() and this one would be in UrlBase.allows_recursion()

Reply:
should we open a separate issue for this?

Reply:
Good idea. Created https://github.com/linkchecker/linkchecker/issues/374.